src/js/lib/monster.util.js

```diff
@@ -81,9 +81,26 @@ define(function(require) {
 		},
 		unformatPhoneNumber: unformatPhoneNumber,
 		unixToDate: unixToDate,
-		updateImagePath: updateImagePath
+		updateImagePath: updateImagePath,
+		isAuthorizedTopicForCrossSiteMessaging: isAuthorizedTopicForCrossSiteMessaging
 	};
 
+	function isAuthorizedTopicForCrossSiteMessaging() {
```

Comment:
It would be better to receive the topic name as a parameter, and check if it is included in the array of topics within this function, based on its name.

Also, do we want this check to be availlable for external parties?

- If so, it will be necessary to add the corresponding documentation file at [docs/monster-util](https://github.com/2600hz/monster-ui/tree/master/docs/monster-util)

- If not, then I would move this function to [core/app.js](https://github.com/2600hz/monster-ui/blob/master/src/apps/core/app.js)